### ISSUE TITLE
Players cant grow rooms

### ISSUE THREAD

**Issue by ccattell** [eccentricdevotion/TARDIS#67]
We are using build 1705 of TARDIS and we are up to date on Spigot builds.  I have tried adding both methods of adding perms to Group Manager

- tardis.room 
- tardis.room.vault

but for some reason players still can not build, its telling them they don't have permission

**Comment by eccentricdevotion** [comment]
Either perm should work.
What method are you using to build, ARS or command?

**Comment by ccattell** [comment]
They have actually tried both ways

 

 

 

 

Charlie Cattell

MSM Technologies

 

Cell: 501.422.8500

Email: charlie@mymsmtech.com

**Comment by ccattell** [comment]
Ok we got the ARS working, apparently we needed the tardis.architectural perm added as well, but he says he is still not able to use the command method

 

 

 

 

Charlie Cattell

MSM Technologies

 

Cell: 501.422.8500

Email: charlie@mymsmtech.com

**Comment by eccentricdevotion** [comment]
Not having any issues growing a vault room with the `tardis.room.vault` permission granted.
Steps I used:

1. Logged into server
2. Used server console to de-op myself
3. Placed dispenser in correct position to grow room
4. Ran the `/tardis room vault` command - _didn't have permission and got the "No permission" message_
5. Used server console to grant myself the perm `tardis.room.vault`
6. Tried the `/tardis room vault` command again - _asked to "click room seed block with TARDIS key"_
7. Clicked block - _room started growing_
